src/ingress.py

```diff
@@ -28,3 +30,16 @@ def __init__(self, charm: ops.CharmBase, key: str, relation_name: str):
             port=jenkins.WEB_PORT,
             strip_prefix=True,
         )
+
```

Comment:
should we add an integration test for this? (traefik in path routing mode)

Reply:
I switched traefik's routing mode to `route` in the corresponding fixture. That should suffice (if route works, subdomain will work too)
```
traefik = await model.deploy(
        "traefik-k8s",
        channel="edge",
        trust=True,
        config={
            "external_hostname": external_hostname,
            "routing_mode": "path",
            "enable_experimental_forward_auth": True,
        },
    )
  
   ```